docs/visual-basic/language-reference/error-messages/name-name-is-not-declared.md

```diff
@@ -27,13 +27,13 @@ A statement refers to a programming element, but the compiler cannot find an ele
 
 6. If you are attempting to compile an SDK-style project (a project with a \*.vbproj file that begins with the line `<Project Sdk="Microsoft.NET.Sdk">`), and the error message refers to a type or member in the Microsoft.VisualBasic.dll assembly, configure your application to compile with a reference to the Visual Basic Runtime Library. By default, a subset of the library is embedded in your assembly in an SDK-style project.
 
-   For example, the following example fails to compile because the <xref:Microsoft.VisualBasic.CompilerServices.Conversions.ToInteger%2A?displayProperty=fullName> method cannot be found. It is not embedded in the subset of the Visual Basic Runtime included with your application.  
+   For example, the following example fails to compile because the <xref:Microsoft.VisualBasic.CompilerServices.Conversions.ChangeType%2A?displayProperty=fullName> method cannot be found. It is not embedded in the subset of the Visual Basic Runtime included with your application.  
 
    [!code-vb[BC30451](~/samples/snippets/visualbasic/language-reference/error-messages/bc30451/program1.vb)]
```

Comment:
```suggestion
   [!code-vb[BC30451](~/samples/snippets/visualbasic/language-reference/error-messages/bc30451/program1.vb?highlight=7)]
```

I just discovered this **highlight** query option. I believe it can be widely adopted in code blocks.

Reply:
Well, then a text file should be put in samples repo beside the vb file to indicate ot is referenced by line number

Reply:
@rpetrusha Is this feature worth the overhead?